### ISSUE TITLE
Support for other parameters like "format"

### ISSUE THREAD

**Issue by 5j9** [mlazowik/templatedata#10]
It seems that there are some parameters that are valid in current version of TemplateData extension but not supported by this tool. Some of them are:
- format 
- example
- autovalue

See https://www.mediawiki.org/wiki/Extension:TemplateData for the complete list and descriptions.

I know, it's been 2 years since last commit and now Mediawiki offers a GUI tool for editing template data, maybe this tool is not going to be maintained anymore, but I preferred the design of this one. In any case, my thanks to the author for their great job, I used to use it a lot.

**Comment by mlazowik** [comment]
You're quite right, I stopped maintaining it because of the “native” editor being finally available. I'll try to estimate how much work would be required to update to current spec. If it is in order of hours, I'll probably do that soon :)

Thanks for the feedback & appreciation!
